Dockerfile.test

```diff
@@ -0,0 +1,26 @@
+FROM 1science/alpine
+
+ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+# get our basic-needs sorted
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories
+RUN apk update
+RUN apk add python3 python3-dev vim bash    \
+                curl      \
+    && curl "https://bootstrap.pypa.io/get-pip.py" | python3 \
```

Comment:
Should we store this file in our own github repo rather than downloading from pypa.io every time?

Reply:
Turns out that pip is installed by either the python3 or python3-dev packages.  So I got rid of the download altogether.